cg/cli/clean.py

```diff
@@ -88,7 +88,7 @@ def hk_alignment_files(
         tag_files = set(housekeeper_api.get_files(bundle=bundle, tags=[tag]))
 
         if not tag_files:
-            LOG.warning(
+            LOG.debug(
```

Comment:
Is this not valuable to have as info?

Reply:
Sort of torn here... The way we use it I feel like it adds nothing, it just bloats our logs

Reply:
Feel free to go ahead with your choice!